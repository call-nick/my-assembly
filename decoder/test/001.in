0x73
0xC4
0xC1
0x81
0x99
0xAA
0xB7