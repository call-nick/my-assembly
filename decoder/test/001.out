MOVI D, 115
OUT A
IN B
ADD A, B
SUB C, B
MUL C, C
DIV B, D
